integration-tests/features/steps/common.py

```diff
@@ -1643,11 +1657,8 @@ def check_github_details_file(context, package, ecosystem):
 
     check_attribute_presence(data, "summary")
     check_attribute_presence(data, "details")
-    check_attribute_presence(data, "schema")
 
-    schema = data["schema"]
-    check_attribute_presence(schema, "name")
-    check_attribute_presence(schema, "version")
+    check_schema_attribute(data, "github_details", "1-0-4")
```

Comment:
Will we need to change this every time we bump schema version in [GithubTask](https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/f8a_worker/workers/githuber.py#L18) ?

(The same applies to all places where we check schema version.)

Reply:
Well, yes (partially because changes in schema means that other tests would need to be updated as well). In the future tests I'd like to check the JSON against real schema specification.